src/store/slice/authSlice.ts

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  isAuth: false,
+};
+
+interface state {
+  isAuth: boolean;
+}
+export const authSlice = createSlice({
+  name: "auth",
+  initialState,
```

Comment:
check localStorage for initialState initialization. So that already loggedIn user has loggedIn State to true.